spirv_cross.cpp

```diff
@@ -1048,7 +1048,7 @@ ShaderResources Compiler::get_shader_resources(const unordered_set<VariableID> *
 	return res;
 }
 
-bool Compiler::type_is_top_level_block(const spirv_cross::SPIRType &type) const
+bool Compiler::type_is_top_level_block(const SPIRType &type) const
```

Comment:
Pushed a fix for this. Silly IDE auto-added it.

Reply:
Okay. I wonder why this PR is not now choking on it?